### PR TITLE
Crawler updates

### PR DESCRIPTION
**Pull request by datagutten**

### CODE REVIEW

File: comics/aggregator/crawler.py [hunk 1]
```diff
@@ -298,3 +298,23 @@ def crawl_helper(self, feature_id, pub_date):
...
                 url = page.src('img[itemprop="image"]')
                 return CrawlerImage(url)
+
+
+class NettserierCrawlerBase(CrawlerBase):
+    """Base comics crawler for all comics posted at nettserier.no"""
+    time_zone = 'Europe/Oslo'
+
+    def crawl_helper(self, short_name, pub_date):
+        page_url = 'https://nettserier.no/%s/' % (short_name)
```
Comment (jodal) [inline comics/aggregator/crawler.py]: Just FYI: The parens around `short_name` here has no effect. To create a one-tuple, you must have a trailing comma, e.g. `(short_name,)`. The `%` formatting operator accepts both single values and tuples, so this works anyway.

File: comics/aggregator/crawler.py [hunk 2]
```diff
@@ -298,3 +298,23 @@ def crawl_helper(self, feature_id, pub_date):
...
+        page = self.parse_page(page_url)
+        site_date = page.text('p[class="comic-pubtime"]')
+        f = 'Published %Y-%m-%d %H:%M:%S'
+        date = datetime.datetime.strptime(site_date, f)
```
Comment (jodal) [inline comics/aggregator/crawler.py]: I think it is better if either:

-`f` is inlined and the line broken in two after `strptime(`, or
- `f` is renamed `format`.

File: comics/comics/ikkesaro.py [hunk 3]
```diff
@@ -0,0 +1,18 @@
+# encoding: utf-8
```
Comment (jodal) [inline comics/comics/ikkesaro.py]: This line is redundant if there is only ASCII characters in the file, i.e. there is no Norwegian letters.

If I recall correctly, the line actually makes the Python interpreter stop reading the file, change mode, and start over again. So in theory it has negative effect on performance.

File: comics/comics/wyyrd.py [hunk 4]
```diff
@@ -1,4 +1,5 @@
-from comics.aggregator.crawler import CrawlerBase, CrawlerImage
+# encoding: utf-8
```
Comment (jodal) [inline comics/comics/wyyrd.py]: This line is redundant too.

File: comics/comics/ikkesaro.py (hunk 3 above)
Comment (datagutten) [inline comics/comics/ikkesaro.py]: Thanks for explaining this. I was wondering why some crawlers had it and someone not.

**Pull request merged**